front-page.php

```diff
@@ -198,19 +221,19 @@
 											<path d="M16.3333 14.0002V12.6669C16.3328 12.0761 16.1362 11.5021 15.7742 11.0351C15.4122 10.5682 14.9053 10.2346 14.3333 10.0869" stroke="#737373" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
 											<path d="M11.6667 2.08691C12.2404 2.23378 12.7488 2.56738 13.1118 3.03512C13.4749 3.50286 13.672 4.07813 13.672 4.67025C13.672 5.26236 13.4749 5.83763 13.1118 6.30537C12.7488 6.77311 12.2404 7.10671 11.6667 7.25358" stroke="#737373" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
 										</svg>
-										<?php print "{$member_count}&nbsp;".__("Members", "community-portal"); ?>
+										<?php echo esc_html( "{$member_count}&nbsp;" ) . esc_html_e( 'Members', 'community-portal' ); ?>
 								</div>
 								<div class="groups__card-info">
 									<div class="groups__card-tags">
-										<?php 
+										<?php
 												$tag_counter = 0;
 										?>
 										<ul class="groups__card-tags__container">
-										<?php foreach($meta['group_tags'] AS $key =>  $value): ?>
-											<li class="groups__tag"><?php print $value; ?></li>
+										<?php foreach ( $meta['group_tags'] as $key => $value ) : ?>
+											<li class="groups__tag"><?php echo esc_html( $value ); ?></li>
 											<?php $tag_counter++; ?>
-											<?php if($tag_counter === 2 && sizeof($meta['group_tags']) > 2): ?>
-												<li class="groups__tag">+ <?php print sizeof($meta['group_tags']) - 2; ?> <?php print __(' more tags', 'community-portal'); ?></li>
+											<?php if ( 2 === $tag_counter && count( $meta['group_tags'] ) > 2 ) : ?>
+												<li class="groups__tag">+ <?php echo esc_html( count( $meta['group_tags'] ) - 2 ); ?> <?php echo esc_html_e( ' more tags', 'community-portal' ); ?></li>
 												<?php break; ?>
```

Comment:
This line throws a warning that code after this break statement can't run

Reply:
Yeah this one is fine.